### ISSUE TITLE
Stackable (speed) potions for kit Scout

### ISSUE THREAD

**Issue by Wxffel** [HGLabor/HGLaborKitAPI2.0#71]
The name says it all
idea by veto

**Comment by copyandexecute** [comment]
stackt es sich denn nicht?

**Comment by copyandexecute** [comment]
bump

**Comment by copyandexecute** [comment]
bump

**Comment by Krxwallo** [comment]
die stacken sich lol warum ist der issue immernoch opened

**Comment by Wxffel** [comment]
> die stacken sich doch lol warum ist der issue immernoch opened

Falsch. Die Speed-Potions werden nicht gestackt. Der Bug besteht noch immer. Getestet in FFA.
Bild: https://i.imgur.com/KLORyrt.png
